File: tuts/001-lightsail-gs/ruby/scenario_getting_started.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require_relative 'lightsail_wrapper'

def run_scenario
  client = Aws::Lightsail::Client.new
  wrapper = LightsailWrapper.new(client)
  puts "Running Lightsail getting started scenario..."
  # TODO: setup, interact, teardown
  puts "Scenario complete."
end

run_scenario if __FILE__ == $PROGRAM_NAME

File: tuts/001-lightsail-gs/python/scenario_getting_started.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import uuid
import boto3
from lightsail_wrapper import LightsailWrapper


def run_scenario():
    client = boto3.client("lightsail")
    wrapper = LightsailWrapper(client)
    suffix = uuid.uuid4().hex[:8]
    print(f"Running Lightsail getting started scenario...")
    # TODO: implement setup, interact, teardown
    print("Scenario complete.")


if __name__ == "__main__":
    run_scenario()
